### PR TITLE
feat(parental-leave): moving the file upload field

### PR DESCRIPTION
**Pull request by Thornvik**
## What

Moving the file upload field to allow it to be used for more than just the upload of self employed attachments

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts [hunk 1]
```diff
@@ -491,6 +463,98 @@ export const ParentalLeaveForm: Form = buildForm({
             }),
           ],
         }),
+        buildSubSection({
+          id: 'fileUpload',
+          title: 'File upload',
```
Comment (karenbjorg) [inline libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts:468]: Can you add the generic title text here

**Review by karenbjorg — COMMENTED** [review]
one comment, otherwise lgtm

File: libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.service.ts [hunk 2]
```diff
@@ -198,17 +203,51 @@ export class ParentalLeaveService {
...
         attachmentBytes: pdf,
       })
+    } else {
+      const files = getValueViaPath(application.answers, 'fileUpload.file')
+
+      if ((files as { file: unknown[] })?.file) {
+        const pdf = await this.getPdfs(application)
+
+        attachments.push({
+          attachmentType: 'other',
```
Comment (qdd1) [inline libs/application/template-api-modules/src/lib/modules/templates/parental-leave/parental-leave.service.ts:247]: Can you add this to constants.ts  too

**Review by karenbjorg — APPROVED** [review]
looks good

**Pull request merged**